draft-ietf-openpgp-pqc.md

```diff
@@ -925,8 +926,7 @@ Two migration strategies are recommended:
    Implementations not understanding PQ(/T) certificates can use the legacy certificate, while PQ(/T)-capable implementations will prefer the newer certificate.
    This allows having an older v4 or v6 certificate for compatibility and a v6 PQ(/T) certificate, at a greater complexity in key distribution.
```

Comment:
```suggestion
   This allows having a traditional certificate for compatibility and a v6 PQ(/T) certificate, at a greater complexity in key distribution.
```
Although not critical maybe an editorial improvement, in order to focus on the terminology "traditional" instead of "old".